meson_options.txt

```diff
@@ -0,0 +1,9 @@
+# -*- mode: meson -*-
+
+option('pkgconfiglibdir', type : 'string', value : '', description : 'directory for standard pkg-config files')
+
+option('systemd', type : 'combo', choices : ['auto', 'true', 'false'], description : 'libsystemd support')
```

Comment:
Not completely sure about the combo type. Do we really want the auto?

Reply:
This is to automatically use these libraries if they are already installed. Otherwise, these features are automatically turned off.

Reply:
We just discussed this here at ALPSS (we = Keith, Hannes and me). So we decided to make libuuid, json-c mandatory and get rig of the systemd dependency by implementing the uuid lookup directly in the library.